support-frontend/assets/pages/contributions-landing/components/ContributionForm.jsx

```diff
@@ -174,9 +171,7 @@ const formHandlersForRecurring = {
 const formHandlers: PaymentMatrix<PropTypes => void> = {
   ONE_OFF: {
     Stripe: (props: PropTypes) => {
-      if (props.stripeElementsTestVariant !== 'stripeCardElement') {
-        openStripePopup(props);
```

Comment:
Do we need to remove this function from the codebase or is it in use elsewhere?

Reply:
good question. It's actually still used by the recurring contributions handler, which still uses stripe checkout. I'm working on removing this in a separate branch